CHANGELOG.md

```diff
@@ -73,6 +75,7 @@ SPDX-License-Identifier: CC-BY-SA-4.0
 - The dark-mode is also applied to the read-only-view and can be toggled from there.
 - Access tokens for the CLI and 3rd-party-clients can be managed in the user profile.
 - Change editor font to "Fira Code"
+- Note tags can be set as yaml-array in frontmatter
```

Comment:
```suggestion
- Note tags can be set as yaml array in frontmatter
```

Reply:
We should be consistent with the translations in this point. What do you prefer?
 - yaml-array
 - yaml-collection
 - yaml array
 - yaml collection

Reply:
s.o.

Reply:
I kept `yaml-array` and changed the other texts to match that.